src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,45 @@
+import cn from 'classnames';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  person: Person;
+  selectedPerson: string;
+};
+
+export const PeoplePage: React.FC<Props> = ({ person, selectedPerson }) => {
```

Comment:
It looks like `Person` or `PersonRow` component, as it's only for one person component